packages/messaging-api-messenger/src/MessengerClient.ts

```diff
@@ -1182,16 +1178,31 @@ export default class MessengerClient {
   /**
    * Retrieving Labels Associated with a PSID
    *
-   * https://developers.facebook.com/docs/messenger-platform/send-messages/broadcast-messages/target-broadcasts#get_all_labels
+   * https://developers.facebook.com/docs/messenger-platform/identity/custom-labels#retrieving_labels_by_psid
    */
-  // FIXME: [type] return type
   getAssociatedLabels(
     userId: string,
     options: { accessToken?: string; fields?: string[] } = {}
-  ) {
+  ): Promise<{
```

Comment:
It's hard to use if the return type is not export to framework user.

Reply:
I'd say it's better than having `any` as return type. We need to put another few days to improve this file.